src/components/JoinUs/Perks.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import { Row, Col } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2 } from '../Typography'
+import Image from '../Common/Image'
+
+import { Section, ShiftedColumn } from './elements'
+
+const PerksListContainer = styled(ShiftedColumn)`
+  display: grid;
+  row-gap: ${remcalc(24)};
+  grid-template-columns: 1fr 1fr;
+  column-gap: 42px;
```

Comment:
If you compare the Zeplin version of the Join Us page:
![image](https://user-images.githubusercontent.com/15656538/51605253-708c0480-1f06-11e9-9581-b5620e79af8b.png)

to coded:
![image](https://user-images.githubusercontent.com/15656538/51605205-45091a00-1f06-11e9-8ff3-20fa4e53751a.png)

you can see that this spacing is not quite as designed

### See Grid documentation in Zeplin
![image](https://user-images.githubusercontent.com/15656538/51605100-f52a5300-1f05-11e9-86c3-4c9f97b76f4e.png)

![image](https://user-images.githubusercontent.com/15656538/51605118-05423280-1f06-11e9-9b0e-684236199173.png)

It's the gutters that are hard-coded to a specific value & then the column widths are dependent on what is left over.

etc. (More breakpoints visible in Zeplin)

---

As much as I love the idea of using CSS Grid!! :heart_eyes:, I think it's worth us deciding on common conventions to use throughout the codebase. Since we don't use CSS Grid anywhere else to achieve our columns & rows, I'm not sure that it's a great idea to be mixing solutions?

To be honest, I think it would make our lives muuuuch easier if we moved to CSS Grid everywhere, but that's a big refactor job & I think consistency is more important than _which_ solution you go with.

But I'm totally open to being wrong on this.

Reply:
not sure what you meant in this comment - seems like a mixture of a new and an old comment? 

Regarding the gutters/column spacing - this is the bug that we've been talking: our whole grid/column system is not 100% accurate to the designs. I don' think it should be fixed in this MR. (If that's not what you were referring, sorry :) )